src/libs2e.c

```diff
@@ -0,0 +1,550 @@
+///
+/// Copyright (C) 2015-2017, Cyberhaven
+/// All rights reserved.
+///
+/// Licensed under the Cyberhaven Research License Agreement.
+///
+
+#include <assert.h>
```

Comment:
It's not clear to me why this file is in the diff.

Reply:
That's odd, I have deleted this file.